src/features/ITGovernance/Admin/GRBReview/GRBReviewForm/Participants/index.tsx

```diff
@@ -1,26 +1,187 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Controller } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { useHistory, useLocation } from 'react-router-dom';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { Button, Fieldset, FormGroup, Grid } from '@trussworks/react-uswds';
+import {
+  GetSystemIntakeGRBReviewDocument,
+  SystemIntakeGRBReviewerFragment,
+  SystemIntakeGRBReviewFragment,
+  SystemIntakeGRBReviewType,
+  useUpdateSystemIntakeGRBReviewFormInputTimeframeAsyncMutation
+} from 'gql/generated/graphql';
 
+import Alert from 'components/Alert';
+import DatePickerFormatted from 'components/DatePickerFormatted';
 import { EasiFormProvider, useEasiForm } from 'components/EasiForm';
+import FieldErrorMsg from 'components/FieldErrorMsg';
+import RequiredAsterisk from 'components/RequiredAsterisk';
 import { GRBReviewFormStepProps } from 'types/grbReview';
+import { SetGRBParticipantsAsyncSchema } from 'validations/grbReviewSchema';
 
+import ParticipantsTable from '../../ParticipantsSection/_components/ParticipantsTable';
 import GRBReviewFormStepWrapper, {
   GRBReviewFormStepSubmit
 } from '../GRBReviewFormStepWrapper';
 
-// TODO: Update fields type
-type ParticipantsFields = {};
+type ParticipantsFields = {
+  grbReviewers: SystemIntakeGRBReviewerFragment[];
+  grbReviewAsyncEndDate: SystemIntakeGRBReviewFragment['grbReviewAsyncEndDate'];
+};
 
 const Participants = ({ grbReview }: GRBReviewFormStepProps) => {
-  const form = useEasiForm<ParticipantsFields>();
+  const { t } = useTranslation('grbReview');
+
+  const history = useHistory();
+  const { pathname } = useLocation();
+  const [startGRB, setStartGRB] = useState(false);
```

Comment:
Hey Gary, I know when we paired, we used react state management, but in hindsight,it may be better use RHF to control this value?  We can always set the default value to false when we instantiate the form.  Then just pass a generic bool prop to the wrapper to signify a final form submission.  We can then call the form `setValue`, which will allow us to determine if that value is dirty.  I'm suggesting this for when we implement the next PR, which we would only want to pass dirty values.  If we keep this state outside the form, we wont be able to do that with this field.  Let me know if you want to sync, as this can be hard to describe